src/main/java/org/example/service/ConnectionService.java

```diff
@@ -37,7 +37,6 @@ public Connection connectToDb() {
             LOGGER.info("Connection Established!");
         } catch (Exception e) {
```

Comment:
Maybe it is better to log the exception message.
Why you catch the generic exception?